src/main/resources/archetype-resources/silo/src/main/resources/application-vagrant.yml

```diff
@@ -0,0 +1,15 @@
+stage: vagrant
+
+spring:
+    datasource:
+        url: jdbc:CHANGEME://${spring.config.name}.CHANGME.service.vagrant.consul/CHANGEME
```

Comment:
that can be removed, why did you add it? Normally the "vagrant" should be replaced by `${stage}`